tools/data_integrity/data_deletion/delete_physiological_file.php

```diff
@@ -181,6 +181,19 @@ function deletePhysiologicalFile($physioFileID, $confirm, $printToSQL, $DB, &$ou
     );
     print_r($electrodes);
 
+    echo "\nCoordinate Systems\n";
+    echo "----------------------------\n";
+    $coordinate_systems = $DB->pselectCol(
+        'SELECT DISTINCT FilePath
+        FROM physiological_coord_system
+        JOIN physiological_coord_system_electrode_rel
+        USING (PhysiologicalCoordSystemID)
+        WHERE PhysiologicalFileID=:pfid
+        AND FilePath IS NOT NULL',
```

Comment:
That is the `not registered` state (e.g. modality). It is used for imported RB data or when we insert a coordinate system that is not registered. Because we want to delete files in that script, there are no files associated with the `not registered`.

Reply:
@regisoc Any update on this one?